caddyhttp/markdown/markdown.go

```diff
@@ -107,6 +107,7 @@ func (md Markdown) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error
 
 		// Set path to found index file
 		fpath = idx
+
```

Comment:
Not a showstopper, but in the future, I prefer that closing curly braces don't have extra blank lines before them, it makes it easier to see exactly what it's enclosing.